newIDE/app/src/EventsFunctionsExtensionsLoader/Storage/BrowserEventsFunctionsExtensionWriter.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import { serializeToJSObject } from '../../Utils/Serializer';
+
+const downloadStringContentAsFile = (
```

Comment:
I assume most of this code is a copy from other platforms right?

Reply:
This is not used, I removed it, good catch!

Reply:
Not sure what you mean with platforms here?

Reply:
No, there is no really look alike code anywhere else in the codebase